gpcc-mocks/src/test/java/uk/nhs/adaptors/gpccmocks/GpccMocksApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package uk.nhs.adaptors.gpccmocks;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class GpccMocksApplicationTests {
```

Comment:
Is this test needed?

Reply:
Provided by Spring Initializr. At the very least is ensure the application can start without errors. Can be removed if needed though.

Reply:
Yeah I'd say it's not needed